topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -480,10 +480,10 @@ The BAM file contains information for all our reads, making it difficult to insp
 >
 > > ### {% icon hands_on %} Hands-on: Check the number of reads mapped to each chromosome
 > >
-> > 1. **Samtools idxstats** {% icon tool %}:
-> >    - {% icon param-files %} *"BAM file"*: `mapped.bam` files (outputs of **RNA STAR** {% icon tool %})
+> > 1. [**Samtools idxstats**](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/samtools_idxstats/samtools_idxstats/2.0.3) {% icon tool %}:
+> >    - {% icon param-files %} *"BAM file"*: `mapped.bam` files (outputs of [**RNA STAR**](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a) {% icon tool %})
```

Comment:
No need for the link here as it is just to help people to identify the input they need to use